lib/rules/web/admin_console/4.2/environments.xyaml

```diff
@@ -38,7 +38,7 @@ add_env_var_value:
 choose_container:
   elements:
   - selector:
-      xpath: //div[@class='dropdown']//span[@class='co-resource-item']
+      xpath: //*[(.='Container:')]/following-sibling::*//button[contains(@class, 'dropdown__toggle')]
```

Comment:
same here

Reply:
same as above, now the element is defined as `//div` but I'm afraid someday it will change to `//label` or something else